lib/tasks/evm_dba.rake

```diff
@@ -117,8 +137,34 @@ namespace :evm do
         opt :table,      "Tablename to reindex (if only perorm on one)", :type => :string
       end
 
-      opts = opts.delete_if { |_, v| v == false }
-      EvmDatabaseOps.gc(opts)
+      args    = {}
+      opts    = opts.delete_if { |_, v| v == false }
+      options = (options[:aggressive] ? GC_AGGRESSIVE_DEFAULTS : GC_DEFAULTS).merge(opts)
+
+      pg_env = {"PGUSER" => options[:username], "PGPASSWORD" => options[:password]}.delete_blanks
+
+      raise "Vacuum requires database" unless options[:dbname]
+
+      vacuum_args = {}
+      vacuum_args[:analyze] = nil             if options[:analyze]
+      vacuum_args[:full]    = nil             if options[:full]
+      vacuum_args[:verbose] = nil             if options[:verbose]
+      vacuum_args[:table]   = options[:table] if options[:table]
+
+      run_command("vacuumdb", opts, args)
+
+      $log.info("Running command... #{AwesomeSpawn.build_command_line("vacuumdb", params)}")
+      output = AwesomeSpawn.run!("vacuumdb", :params => params, :env => pg_env).output
```

Comment:
Where is vacuum_args used?

```suggestion
      output = AwesomeSpawn.run!("vacuumdb", :params => vacuum_args, :env => pg_env).output
```

Reply:
Yes, copy-pasta mistake from combining multiple methods into one (from `PostgresAdmin`).  Also did the same below with `reindex_args`.

Will fix.

Reply:
Fixed.